### DISCUSSION TITLE
Error deploying

### DISCUSSION THREAD

**Discussion by digital911** [KelvinTegelaar/CIPP#2323] [Q&A]
Hi, when deploying to Azure I keep getting the following two errors:

**The content server has rejected the request with: BadRequest
Reason: No matching Static Web App was found or the api key was invalid.**

I have deleted the resource group, verified my fork URLs and verified my personal access token. I have even tried a different region on the approved list to no avail.

Does anyone have any advice?

**Comment by KelvinTegelaar** [comment]
Hi Digital,

Support is only available for sponsors - if you're a sponsor you can contact our helpdesk.
